charts/kspm-collector/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: kspm-collector
 description: Sysdig KSPM collector
 
-version: 0.9.3
-appVersion: 1.35.0
+version: 0.9.4
```

Comment:
I suppose this should be a minor and not a patch release

Reply:
@mavimo this is actually just a change to the way an existing ENV flag is enabled. It doesn't add / remove any new functionality.

Reply:
@zohar-arad mmm, ok, do you know why the appVersion move from 1.35.0 to 1.36.0 then?

Reply:
@mavimo sorry - that's my bad. Let me revert it